.env.dist

```diff
@@ -20,10 +20,15 @@ REG_WS_CLIENT_ID=
 REG_WS_SECRET=
 # Bruincast configs
 HOST=bclive.oid.ucla.edu
-SECRET=***Ask Rex or Max***
+SECRET=******
 TOKEN_NAME=wowzatoken
 VALIDITY=90000
 BRUINCAST_API_LOGIN=
 BRUINCAST_API_USERNAME=
 BRUINCAST_API_PASSWORD=
-BRUINCAST_API=
\ No newline at end of file
+BRUINCAST_API=
+# Video reserves FTP services.
+VIDEORES_FTP_HOST=164.67.141.31
```

Comment:
@maxpluez Leave out the HOST and FILEPATH as well. That can be considered potentially sensitive info